packages/docs/locales/en/en.json

```diff
@@ -57,6 +57,46 @@
         "label": "Replaces the default label."
       }
     },
+    "VaModal": {
+      "props": {
+        "value": "The current visibility state of the modal",
+        "position": "!!! Not implemented yet !!!",
```

Comment:
shouldn't be in docs then?

Reply:
@asvae It's being parsed from the component, but it doesn't work as intended yet